app/src/main/java/fr/free/nrw/commons/kvstore/BasicKvStore.java

```diff
@@ -75,6 +75,11 @@ public String getString(String key) {
         return getString(key, null);
     }
 
+    @Override
```

Comment:
No need to add `Set<String>` in `BasicKvStore`. 

`JsonKVStore` already supports it. 

```
Type setType = new TypeToken<Set<String>>() {}.getType();
directKvStore.putJson("keyName", new HashSet<String>());
directKvStore.getJson("keyName", setType);

```

Reply:
Have updated our Wiki: 

https://github.com/commons-app/apps-android-commons/wiki/Code-style#shared-preferences

Reply:
Hello @maskaravivek 
I need some help.
When I use getJson("keyName", setType) I get error "java.util.HashSet cannot be cast to java.lang.String". That is because MultiSelectListPreference are stored as HashSet<String> in SharedPreferences and not as json string. Should I somewhere explicitly put it into store in json string format?

Reply:
Hope I got it right.